### PR TITLE
evm: Pass value param to the txm

### PR DESCRIPTION
**Pull request by nickcorin**

### CODE REVIEW

File: core/services/relay/evm/chain_writer.go [hunk 1]
```diff
@@ -111,6 +111,11 @@ func (w *chainWriter) SubmitTransaction(ctx context.Context, contract, method st
 		checker.CheckerType = txmgrtypes.TransmitCheckerType(methodConfig.Checker)
 	}
 
+	v := big.NewInt(0)
```
Comment (ilija42) [inline core/services/relay/evm/chain_writer.go:114]: why is txmgr accepting big.Int without a pointer  😫

**Review by ilija42 — DISMISSED** [review]

File: core/services/relay/evm/chain_writer.go (hunk 1 above)
Comment (nickcorin) [reply core/services/relay/evm/chain_writer.go:114]: We're busy discussing whether it's in scope to edit this too along with the other `big.Int` -> `*big.Int` changes we're making.

Comment (nickcorin) [reply core/services/relay/evm/chain_writer.go:114]: But there is an argument that `nil` is not a valid tx value and using `big.Int` helps prevent accidental panics at the cost of being a little ugly.

**Review by ilija42 — APPROVED** [review]

**Review by silaslenihan — APPROVED** [review]
LGTM!

**Review by patrick-dowell — APPROVED** [review]

**Pull request merged**